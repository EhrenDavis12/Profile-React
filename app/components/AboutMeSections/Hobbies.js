// import PropTypes from 'prop-types';
import React from 'react';
import Banner from 'components/Banners/Banner';
// import { P } from 'components/SharedStyles/styles';

function Hobbies() {
  return (
    <div className="container">
      <Banner>
        <ul>
          <li>Programming - Go to work and program, Go home and program</li>
          <li>Dancing - West Coast Swing - any chance I get</li>
          <li>Hiking - A good way to clear your mind and find your ground</li>
          <li>
            Billiards - Was in a league and casually shooting with friends
          </li>
          <li>Ultimate Frisbee - Saturday pick up games</li>
          <li>ATV Riding - Off road here I!</li>
        </ul>
      </Banner>
    </div>
  );
}

export default Hobbies;
