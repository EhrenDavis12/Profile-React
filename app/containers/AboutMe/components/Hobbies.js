// import PropTypes from 'prop-types';
import React from 'react';
// import HeaderBar from 'components/HeaderBar';
import { P, H2 } from './styles';

function Hobbies() {
  return (
    <div className="row">
      <div className="row col-lg-12">
        <H2>Hobbies</H2>
      </div>
      <div className="row col-lg-12">
        <P>
          Experienced Software Engineer with a demonstrated history of working
          in the computer software industry. Strong engineering professional
          skilled in Bash scripting, Automation tools, and Problem solving.
        </P>
        <P>
          Software Engineer II at LawLogix Group, Inc.
          <br />
          Employed: May 2013 – PresentEmployment
          <br />
          located in Phoenix, AZ 85012
          <br />
        </P>
        <P>
          Responsible for re-engineering, maintaining and error handling of an
          automated file batch upload system running daily transaction for 80+
          different clients. Reduced implementation by 60% and proactively
          decreased errors and client inquiries by 30%. Responsible for ensuring
          the proper handling of client sensitive information (PII). Review,
          Design, improve, and automate client data import process. Improved
          data imported accuracy by 42% and decreased steps by 33%. Utilized
          SQL, system logs, and created additional quality assurance
          techniques/tools and steps to ensure validations in identifying issues
          or answering client questions within same day SLA. Developed
          enhancements to the code base based on client’s SOW requirements.
        </P>
      </div>
    </div>
  );
}

export default Hobbies;
