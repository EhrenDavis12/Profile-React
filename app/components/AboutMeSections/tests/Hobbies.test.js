import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';
import Hobbies from '../Hobbies';

describe('<Hobbies />', () => {
  it('should render the page', () => {
    const renderedComponent = shallow(<Hobbies />);
    expect(renderedComponent).toMatchSnapshot();
  });
});
