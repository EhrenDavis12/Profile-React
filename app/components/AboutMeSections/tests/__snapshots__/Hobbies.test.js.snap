// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hobbies /> should render the page 1`] = `
<div
  className="container"
>
  <BannerSection>
    <ul>
      <li>
        Programming - Go to work and program, Go home and program
      </li>
      <li>
        Dancing - West Coast Swing - any chance I get
      </li>
      <li>
        Hiking - A good way to clear your mind and find your ground
      </li>
      <li>
        Billiards - Was in a league and casually shooting with friends
      </li>
      <li>
        Ultimate Frisbee - Saturday pick up games
      </li>
      <li>
        ATV Riding - Off road here I!
      </li>
    </ul>
  </BannerSection>
</div>
`;
